SemanticSearchV2.md

```diff
@@ -149,7 +145,9 @@ Parameters (query/form):
 -	user_id (optional) : user id for which to look up personalized overrides. Defaults to null. 
 -	relationships_threshold : the mimimum weight for a relationship entry to be added as enrichment. Can be used to prune the result size. Defaults to 0.5. Allowed values are any number between 0 and 1.
 -	document : the binary document to parse as base64 encoded string (according to RFC 3548/4648).
--	type : the type of document either "JOB" or "RESUME".
+-	type : the type of document is either "JOB" or "RESUME".
+-	job_title (optional, only valid when type is JOB) : a pre-defined job title which overrides the job title extracted from a job document. Notice that the job title may also be cleaned in order to get enrichments.
```

Comment:
"Note that..." makes more sense than "Notice that..." in this context. And is "may" the right word here? (What are the conditions under which the title will / won't be cleaned?) I think it's more correct to say "will be cleaned before being used for enrichments." You may also want to include a sentence explaining which field gets overridden in the response when this value is provided. (Same for `location` on the next line)